scripts/spm-integration-test.sh

```diff
@@ -63,13 +89,19 @@ validate_service_metrics() {
     echo "Metric datapoints found for service '$service': " "${metric_points[@]}"
     # Check that atleast some values are non-zero after the threshold
     local non_zero_count=0
-    local expected_non_zero_count=3
+    local expected_non_zero_count=5
     local zero_count=0
-    local expected_max_zero_count=3
+    local expected_max_zero_count=4
+    local flag=false # Becomes true after first non-zero value 
```

Comment:
let's not introduce this change yet, as it's failing the test.

Reply:
Done.